Source/Header/DayHeaderView.swift

```diff
@@ -4,7 +4,13 @@ import DateToolsSwift
 
 public final class DayHeaderView: UIView, DaySelectorDelegate, DayViewStateUpdating, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
   public private(set) var daysInWeek = 7
-  public let calendar: Calendar
+  public var calendar: Calendar {
+    didSet {
+      daySymbolsView.calendar = calendar
+      swipeLabelView.calendar = calendar
+      configure() // TODO: Should I do it that way or better just to set a new vc without adding subview (like in state#didSet)?
```

Comment:
I suggest refactoring the `configure` method and extracting all of the code that should run only once to a separate method (which also calls `configure`). The new `configure` method should be able to be run multiple times during the lifecycle of the view.